### DISCUSSION TITLE
undefined method excerpt_separator in gh pages

### DISCUSSION THREAD

**Discussion by AndrewWPhillips** [mmistakes/minimal-mistakes#4690] [General]
My github pages blog with your theme has been working beautifully for a while now but I recently pushed changes and realised that it had not been publishing for 3 pushes.  So I reverted all my changes and pushed back the last one that published to no avail. 

Eventually I tracked it down to this in the publish log:

github-pages 229 | Error: undefined method `excerpt_separator' for #<Jekyll::Page @name="about.md"> Remote Theme: Cleaning up /tmp/jekyll-remote-theme-20240211-8-9cnfhc 

Has something changed in the theme to break this? Or is it something wrong with github pages?  The repo is exactly the same as it was last week when it published fine.

**Comment by AndrewWPhillips** [comment]
I think it's due to (or triggered by) a recent update to github pages Jekyll 3.9.3 -> 3.9.4.
https://pages.github.com/versions/

BTW I don't use the "excerpt_separator" header but I do use "excerpt" with \<br/\>

**Comment by adamdjbrett** [comment]
I am having the same issue with
https://github.com/adamdjbrett/doctrineofdiscovery.org/
